apps/chef_test/src/chef_test_db_helper.erl

```diff
@@ -57,6 +57,10 @@ start_db(Config, DbName) ->
              "--db-port", PortStr, " --db-host localhost deploy"],
             ["psql -p", PortStr, DbName, " -h localhost <", Schema]
            ],
+    %% FIXME: we should fail explicitely here if any of the commands above
+    %% fail
+    %% FIXME: we're never stopping the pgsql instance we launch here. We should
+    %% see to that
```

Comment:
If you use chef_test_helper:stop_server in your suite cleanup, it will take care of this.